lib/templates.js

```diff
@@ -26,28 +26,28 @@ const fReadORDExtensions = (srv) =>
  * @returns {Array} An array containing paths and it's kind.
  */
 
-const fGeneratePaths = (srv, srvDefinition) => {
-  const srvObj = { name: srv, definition: srvDefinition };
-  const protocols = cds.service.protocols;
+const generatePaths = (srv, srvDefinition) => {
```

Comment:
This is a private function, right? Maybe to name it `_generatePath`?
Also, when defining functions, on some places we use `function` and on the others `const`. Do we follow some rule when to use what? I couldn't find any regularity, it seems it was randomly/arbitrarily decided.

Reply:
As https://github.com/cap-js/ord/issues/57 discussed, we should use function